advanced_source/cpp_export.rst

```diff
@@ -245,7 +245,7 @@ We can now run the following commands to build the application from within the
 
   mkdir build
   cd build
-  cmake -DCMAKE_PREFIX_PATH=/path/to/libtorch ..
+  cmake -S ~/example-app -DCMAKE_PREFIX_PATH=/path/to/libtorch ..
```

Comment:
I think we can't necessarily assume that the example-app is in the user's homedir right? cmake should automatically detect the source dir since you are passing it `..`.

When you ran the tutorial, did you make your `build` directory inside the `example-app/` directory?